docs/cudf/source/api_docs/io.rst

```diff
@@ -43,8 +49,6 @@ ORC
    read_orc
    DataFrame.to_orc
 
-.. currentmodule:: cudf
```

Comment:
was `read_hdf` able to render with this deletion?

Reply:
We have to reset the current module once after line 23, which sets `.. currentmodule:: cudf.io.json` before the JSON reader. The `.. currentmodule:: cudf` is reset on line 27 (after JSON, before Parquet) and applies to the remainder of the file. I checked and this does render correctly.

Reply:
Cool